demo/src/actions.ts

```diff
@@ -1,4 +1,4 @@
-import '@grafana/agent-web/dist/globals';
+import '@grafana/agent-web/build/cjs/globals';
```

Comment:
This looks ugly :-/

Reply:
fixed, `import '@grafana/agent-web/globals';` :)